CorsixTH/Lua/entities/machine.lua

```diff
@@ -255,6 +255,28 @@ function Machine:createHandymanActions(handyman)
     :format(self.object_type.name))
 end
 
+--! Replace this machine (make it pretend it's brand new)
+function Machine:machineReplaced()
+  -- Reset usage stats
+  self.total_usage = 0
+  self.times_used = 0
+  
+  -- Update strength to match the current level of research for it
+  local strength = self.hospital.research.research_progress[self.object_type].start_strength
```

Comment:
I don't think you use strength anywhere except the next line. You might as well just have ```self.strength = self.hospital...```